rootex/framework/systems/post_process_system.cpp

```diff
@@ -20,3 +20,8 @@ void PostProcessSystem::update(float deltaMilliseconds)
 	RenderingDevice::GetSingleton()->unbindSRVs();
 	RenderingDevice::GetSingleton()->setOffScreenRTVDSV();
 }
+
+void PostProcessSystem::addCustomPostProcessing(const String& path)
+{
+	m_Processor.m_PostProcesses.emplace_back(new CustomPostProcess(path));
```

Comment:
add a check to run this line only if the `CustomPostProcess` object for this path has not been created yet, currently, I can create 2 objects for the same path.

Reply:
fixed